integration_tests/src/main/python/iceberg/__init__.py

```diff
@@ -122,4 +123,17 @@ def change_table(spark: SparkSession):
 
     with_cpu_session(change_table,
                      conf = {"spark.sql.parquet.datetimeRebaseModeInWrite": "CORRECTED",
-                             "spark.sql.parquet.int96RebaseModeInWrite": "CORRECTED"})
\ No newline at end of file
+                         "spark.sql.parquet.int96RebaseModeInWrite": "CORRECTED"})
+
+def get_iceberg_s3tables_ns() -> Optional[str]:
```

Comment:
cc @gerashegalov @razajafri should we add s3table dep directly in our integration_tests module?

Reply:
In another context we need a CI running integration and performance regression tests against AWS EMR. We should keep this in mind to make AWS config reusable